Fix "Could NOT find Git (missing: GIT_EXECUTABLE)", git is not needed for releases
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -84,9 +84,7 @@ include(getopt)
 # CK config file to record supported datatypes, etc.
 configure_file(include/ck/config.h.in ${CMAKE_CURRENT_BINARY_DIR}/include/ck/config.h)
 
-# CK version file to record release version as well as git commit hash
-find_package(Git REQUIRED)
-execute_process(COMMAND "${GIT_EXECUTABLE}" rev-parse HEAD OUTPUT_VARIABLE COMMIT_ID OUTPUT_STRIP_TRAILING_WHITESPACE)
+# CK version file to record release version
 configure_file(include/ck/version.h.in ${CMAKE_CURRENT_BINARY_DIR}/include/ck/version.h)
 
 set(ROCM_SYMLINK_LIBS OFF)
